_docs/tasks/traffic-management-v1alpha3/ingress.md

```diff
@@ -9,17 +9,11 @@ redirect_from: /docs/tasks/ingress.html
 {% include home.html %}
 
 In a Kubernetes environment, the [Kubernetes Ingress Resource](https://kubernetes.io/docs/concepts/services-networking/ingress/)
-allows users to specify services that should be exposed outside the cluster.
-For traffic entering an Istio service mesh, however, an Istio-aware [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-controllers)
-is needed to allow Istio features, for example, monitoring and route rules, to be applied to traffic entering the cluster.
+allows specifying services that should be exposed outside the cluster.
 
-Istio provides an envoy-based ingress controller that implements very limited support for standard Kubernetes `Ingress` resources
-as well as full support for an alternative specification,
-[Istio Gateway]({{home}}/docs/reference/config/istio.networking.v1alpha3.html#Gateway).
-Using a `Gateway` is the recommended approach for configuring ingress traffic for Istio services.
-It is significantly more functional, not to mention the only option for non-Kubernetes environments.
+In Istio Service mesh, on top of Kubernetes and other environments, a different concept is applied, namely [Istio Gateway]({{home}}/docs/reference/config/istio.networking.v1alpha3.html#Gateway). It allows Istio features, for example, monitoring and route rules, to be applied to traffic entering the cluster.
```

Comment:
In an Istio service mesh, a better approach (which also works in both Kubernetes and other environments) is to use a different configuration model, namely ...